src/Lykke.Logs/Obsolete/ILogPersistenceManager.cs

```diff
@@ -1,7 +1,9 @@
+using System;
 using System.Collections.Generic;
 
 namespace Lykke.Logs
 {
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
See above